### ISSUE TITLE
Give more feedback abilities

### ISSUE THREAD

**Issue by Tbaut** [paritytech/fether#347]
Not everyone has GitHub to give us feedback.
We could change the "Feedback" link to a modal showing different options:
- Link to Github
- Link to Twitter
- And email (fether-feedback at parity.io)?

**Comment by yjkimjunior** [comment]
@Tbaut I mocked up what this might look like, but opted against the modal as I think it'd be too obstructive. What do you think?

Feedback Options Closed             |  Feedback Options Open
:-------------------------:|:-------------------------:
![feedback closed](https://user-images.githubusercontent.com/10432070/52189377-57bf0f80-2873-11e9-8d24-3b2743513bd7.png) |  ![feedback open](https://user-images.githubusercontent.com/10432070/52189375-57bf0f80-2873-11e9-916a-3e87a0b9bd97.png)

**Comment by Tbaut** [comment]
Sure thanks for the proposal :+1: 
- I'd just like to keep the colors we already use for other menus. e.g white background with shadow.
- keep the corner radius at the top

What do you think?

Also for the email I think we should have the same behaviour as for when we copy the Ethereum address with a tool tip reading "copy email" and then "copied" on click.

![image](https://user-images.githubusercontent.com/33178835/52943366-ebcac400-33a7-11e9-9e3f-8564e7d61c62.png)

**Comment by yjkimjunior** [comment]
@Tbaut I'm more on board with your version :)

**Comment by yjkimjunior** [comment]
should it be a copy email rather than a `mailto:`?

**Comment by Tbaut** [comment]
Do you think `mailto:` is still a thing?
On the OS I've used in the past years it keeps trying to open whatever email client is installed (but mostly unused) on the machine.. and you end up spending time trying to close this annoying thunderbird/outlook/mail onboarding window that just appeared rather than actually writing the email you wanted to write. What's your take on that?

TBH I'm not entirely convinced by my mockup, it feels a bit clunky and not really easy to implement.. I've tried several designs but couldn't bring anything better to the table..

**Comment by axelchalon** [comment]
It could also be a modal when clicking "Feedback", and the user could choose Github/Twitter/Email.

Ideally though I think we should show a form and the user can write their feedback directly and submit it. Seems to me like the path of least friction. Some solutions should exist to easily set up the backend for this. Might not be a priority.

**Comment by Tbaut** [comment]
>It could also be a modal when clicking "Feedback", and the user could choose Github/Twitter/Email.

Yes, this was my original idea as it's quite simple to implement. The message form is indeed prob. a little too much of a hassle.
